### PR TITLE
feat: ✨ password change UI

### PR DESCRIPTION
**Pull request by slugb0t**
The password that auto fills in develop has been changed to match the same password created by pytest.

### CODE REVIEW

File: src/views/account/UserProfile.vue [hunk 1]
```diff
@@ -17,11 +18,64 @@ const userProfile = ref<UserProfile>({
...
+      }
+      // Validate value has at least 1 number
+      if (!/[0-9]/.test(value)) {
+        return new Error("Password must contain at least 1 number");
+      }
+      // Validate value has at least 1 special character
+      if (!/[!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]/.test(value)) {
+        return new Error("Password must contain at least 1 special character");
+      }
+      return true;
```
Comment (megasanjay) [inline src/views/account/UserProfile.vue:69]: Add a validator rule to ensure that the new password is not the same as the old password

File: src/views/account/UserProfile.vue [hunk 2]
```diff
@@ -75,6 +129,31 @@ const updateProfile = (e: MouseEvent) => {
...
+      const response = await fetch(`${baseURL}/auth/password/change`, {
+        body: JSON.stringify(passwordForm.value),
+        method: "POST",
+      });
+
+      loading.value = false;
+      if (!response.ok) {
+        push.error("Something went wrong, please try again later");
+        throw new Error("Password error");
+      }
```
Comment (megasanjay) [inline src/views/account/UserProfile.vue:148]: logout on a successful password change. You can just navigate to /auth/logout for this

Comment (megasanjay) [reply src/views/account/UserProfile.vue:148]: Maybe add a notification/message and then delay the navigation to logout for about 1.5 seconds

**Review by megasanjay — CHANGES_REQUESTED** [review]
Added comments

**Review by megasanjay — APPROVED** [review]

**Comment by megasanjay** [comment]
@slugb0t Can you double check on this conflict

**Comment by slugb0t** [comment]
> @slugb0t Can you double check on this conflict

Conflict resolved 👍🏽

**Review by megasanjay — APPROVED** [review]

**Pull request merged**